src/views/CreateFamily.vue

```diff
@@ -25,12 +24,14 @@
             label="Email Address"
             placeholder="Enter member's email"
             v-model="currentEmail"
+            :error="errorType === 'email'"
           />
           <img
             alt="Add"
             class="mt-6"
             data-cy="add-member"
             src="@/assets/images/PlusIcon.svg"
+            :class="isEmailInValidState ? 'visible' : 'hidden'"
```

Comment:
This is fine to leave as is, but we could re-write it with class bindings in case there are going to be more class name properties in the future
```suggestion
            :class="{ visible: isEmailInValidState, hidden: !isEmailInValidState }"
```

Reply:
I don't think it's necessary at the moment